pages/api/program/[id].ts

```diff
@@ -0,0 +1,54 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { ResponseUtil } from "@utils/responseUtil";
+import { getProgram, deleteProgram } from "@database/program";
+
+/**
+ * handle takes the programId parameter and returns
+ * the program associated with the programId
+ * @param req API request object
+ * @param res API response object
+ */
+export default async function handle(
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> {
+    if (req.method == "GET") {
+        // Obtain program id
+        const { id } = req.query;
+        // Obtain program with provided programId
+        const program = await getProgram(id as string);
```

Comment:
add type for `program`

Reply:
this was something in the GET endpoint that was just merged in, can we add an issue and fix it separately?